### PR TITLE
docs overview: Initial content

### PR DESCRIPTION
**Pull request by sympatheticmoose**
Next steps - fix TODO links for more information and add how does it work section

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -3,29 +3,37 @@ title: "Overview"
...
-Image explaing the Appsody flow..?
+### Appsody Stacks
+These are configurable technology stacks built with popular runtimes and frameworks, such as Java with Eclipse Microprofile and Node.js with Express, that provide the foundation for building applications which can be deployed and managed effectively in Kubernetes. Stacks allow for rapid development, whilst giving the stack provider the ability to control how the applications are composed, for example which security policies are applied, or which version of a dependency is pulled.
 
-### Platform architects..
+[More info on Appsody Stacks](link)
 
-Platform architect's role (how they can utilize Appsody)
+### Appsody Hub
+The central point of control for Appsody Stacks. This is where you can find available stacks, create new stacks or modify existing ones. You can utilise what's there in the public repo, and/or clone to provide a private/interal Hub based on your requirements. By making changes to the Stacks in the Hub, you are able to mass-apply updates to applications which have been built on these, simply by restarting the application.
```
Comment (helenmasters) [inline docs/overview.md]: We've been using American English, so "You can utilize what's in the public repo,..." (I don't think the word there was needed).
private/interal -> private/internal

File: docs/overview.md [hunk 2]
```diff
@@ -3,29 +3,37 @@ title: "Overview"
...
 
-### Developers..
+[More info on Appsody Hub](link)
 
-Developers role (how they can utilize Appsody)
-(Link to slack)
+### Appsody CLI
+No project is complete without a nice new CLI to play with. The Appsody CLI is powerful and intiutive, and allows developers to discover available stacks, bring them into their local development environment, and build/run/test/deploy locally. The Docker container for your application built as a result, can integrated with Tekton pipelines as described [here](TODO) and deployed to Kubernetes cloud environements.
```
Comment (helenmasters) [inline docs/overview.md]: intiutive -> intuitive

Comment (helenmasters) [reply docs/overview.md]: The phrase "built as a result" doesn't seem to fit in the sentence, can we drop it? and make...

"The Docker container, which is built for your application, can be integrated with Tekton pipelines as described [here](TODO) and deployed to Kubernetes cloud environments."

File: docs/overview.md [hunk 3]
```diff
@@ -3,29 +3,37 @@ title: "Overview"
...
+
+## How does it work?
+[TODO CHRIS / Neeraj] Describe common workflow/controller
+
+
+## Contributing
+
+We welcome all contributions, see [CONTRIBUTING](../CONTRIBUTING) and come chat to us in [Slack](if you'd like to get involved).
+
+Want to see your framework available as an Appsody Stack? See [Link](here) and join the #content-providers channel to introduce yourself!
```
Comment (helenmasters) [inline docs/overview.md]: Sounds good... do we need to add the words "on Slack" to the "join the #content-providers channel"?

**Comment by sympatheticmoose** [comment]
thanks for the review @helenmasters  - i'll make those changes

**Pull request merged**